### PR TITLE
Refactor delete functionality using Ajax

### PR DESCRIPTION
**Pull request by agnesa23**
image

### CODE REVIEW

File: Jobsway2goMvc/wwwroot/js/confirm-modal.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿function getDefaultToastrOptions () {
+    return {
+        timeOut: 5000,
+        positionClass: "toast-bottom-left"
+    }
```
Comment (dceneje) [inline Jobsway2goMvc/wwwroot/js/confirm-modal.js]: change position to top right

File: Jobsway2goMvc/Views/Collections/Index.cshtml [hunk 2]
```diff
@@ -14,25 +14,118 @@
...
+                        type: "POST",
+                        url: url,
+                        success: function(data) {
+                                    console.log('response data ', data);
+                                  //if(data.status=="true")
+                                  //{
+                                    toastr.success(message, title);
+                                    window.location.href = '@Url.Action("Index","Collections")';
+                                  //}
+
```
Comment (dceneje) [inline Jobsway2goMvc/Views/Collections/Index.cshtml]: delete comments

File: Jobsway2goMvc/Views/Collections/Index.cshtml [hunk 3]
```diff
@@ -14,25 +14,118 @@
...
+<script>
+
+//Show The Popup Modal For Delete Confirmation
+    //$(function () {
+    
+        @*$('#deleteAction').click(function (e) {
+            e.preventDefault();
+            DeleteRecord();
+        });*@
+    //});
```
Comment (dceneje) [inline Jobsway2goMvc/Views/Collections/Index.cshtml]: delete comments

File: Jobsway2goMvc/Controllers/JobCategoriesController.cs [hunk 4]
```diff
@@ -130,26 +130,8 @@ public async Task<IActionResult> Edit(int id, JobCategory jobCategory)
...
-            {
-                return NotFound();
-            }
-
-            return View(jobCategory);
-        }
-
-        [HttpPost, ActionName("Delete")]
-        [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DeleteConfirmed(int id)
```
Comment (dceneje) [inline Jobsway2goMvc/Controllers/JobCategoriesController.cs]: return anti forgery token attribute back

File: Jobsway2goMvc/Controllers/GroupsController.cs [hunk 5]
```diff
@@ -616,9 +599,8 @@ public async Task<IActionResult> UpdatePrivacy([FromRoute] int id, [FromBody] bo
             return Ok(@group);
         }
 
-        [HttpPost, ActionName("Delete")]
-        [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DeleteConfirmed(int id)
+        [HttpPost]
+        public async Task<IActionResult> DeleteConfirmed([FromRoute] int id)
```
Comment (dceneje) [inline Jobsway2goMvc/Controllers/GroupsController.cs]: return attributes back

File: Jobsway2goMvc/Controllers/CollectionsController.cs [hunk 6]
```diff
@@ -176,29 +176,12 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,Name")] Collection colle
...
-            return View(collection);
-        }
+        
+        
 
         // POST: Collections/Delete/5
-        [HttpPost, ActionName("Delete")]
-        [ValidateAntiForgeryToken]
-        public async Task<IActionResult> DeleteConfirmed(int id)
+        [HttpPost]
```
Comment (dceneje) [inline Jobsway2goMvc/Controllers/CollectionsController.cs]: return attributes back

**Review by dceneje — CHANGES_REQUESTED** [review]

File: Jobsway2goMvc/wwwroot/js/confirm-modal.js (hunk 1 above)
Comment (agnesa23) [reply Jobsway2goMvc/wwwroot/js/confirm-modal.js]: done

File: Jobsway2goMvc/Views/Collections/Index.cshtml (hunk 2 above)
Comment (agnesa23) [reply Jobsway2goMvc/Views/Collections/Index.cshtml]: deleted

File: Jobsway2goMvc/Controllers/GroupsController.cs (hunk 5 above)
Comment (agnesa23) [reply Jobsway2goMvc/Controllers/GroupsController.cs]: added attribute back

File: Jobsway2goMvc/Views/Collections/Index.cshtml [hunk 7]
```diff
@@ -37,95 +37,4 @@
...
-            event.preventDefault();
-            event.stopPropagation();
-            callback(true);
-            confirm.modal('hide');
-
-        });
-    
-    }*@
-    
-</script>
```
Comment (dceneje) [inline Jobsway2goMvc/Views/Collections/Index.cshtml]: Was this intended to be deleted?

**Review by dceneje — CHANGES_REQUESTED** [review]

**Review by dceneje — APPROVED** [review]

**Pull request merged**